language-server/modules/langserver-core/src/test/resources/codeaction/update-documentation/source/updateDocumentation3.bal

```diff
@@ -0,0 +1,9 @@
+
+# This function has a doc with multiple lines. Once updated, it should
+# preserve these lines
+#
```

Comment:
Are these relevant to this particular test?

Reply:
These are okay.

Reply:
@MaryamZi Yes. This test case is to test the ability to retain multiline descriptions when `update documentation` code action is being used.